cli/cli.js

```diff
@@ -130,8 +135,17 @@ function traverseFiles(file) {
 function traverseNode(node, basePath) {
   switch (node.type) {
     case 'ImportDeclaration':
-      const isRelativePath = node.source.value.startsWith('.');
-      const importPath = path.resolve(isRelativePath ? basePath : NODE_PATH, node.source.value);
+      let filepath = node.source.value
+      let dirpath = NODE_PATH
+      if (flags.rootAlias && filepath.indexOf(flags.rootAlias) === 0) {
+        filepath = filepath.replace(/^~\/?/, '')
+        dirpath = flags.root
+      }
+      if (filepath.startsWith('.')) {
```

Comment:
What do you think about to keep the `isRelativePath` variable for a better understanding?
https://github.com/andreruffert/i18n-literally/pull/8/files#diff-d72e128c30ac15619754c48d8662b401L133